### PR TITLE
Updates README.md

### PR DESCRIPTION
**Pull request by mchmielarz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,28 @@
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.assertj/assertj-vavr/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.assertj/assertj-vavr)
+
 # assertj-vavr
-Assertions for Vavr - http://www.vavr.io/
+Assertions for [Vavr](http://www.vavr.io/)
```
Comment (Opalo) [inline README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+
+## Usage
+
+### Dependency in your project:
+- build.gradle:
+```groovy
```
Comment (Opalo) [inline README.md]: Why `groovy` here?

Comment (mchmielarz) [reply README.md]: build.gradle is usually created in groovy, right? Maybe it is superfluous

Comment (Opalo) [reply README.md]: No, it's all good :) I just misunderstood it.

**Pull request merged**